### ISSUE TITLE
`DockerizedIBGateway` container reference typo

### ISSUE THREAD

**Issue by Xepheryy** [nautechsystems/nautilus_trader#1889]
# Bug Report

### Expected Behavior
The string in the `DockerizedIBGateway.IMAGE` will allow us to pull the correct container from [here](https://github.com/gnzsnz/ib-gateway-docker/pkgs/container/ib-gateway)

### Actual Behavior
It attempts to pull a container that does not exist as the tag [stable] is not attached to any container image.

### Steps to Reproduce the Problem

1. Set up per-requisites required to run the `DockerizedIBGateway`
2. Attempt to run the code that calls the initial set up of the container as outlined in the documentation for IBKR integration

### Specifications

- OS platform: Linux Mint 
- Python version: 3.10.12
- `nautilus_trader` version: 1.199.0 Beta

**Comment by cjdsellers** [comment]
Hi @Xepheryy

Thanks for the report.

Any thoughts here @rsmb7z or @benjaminsingleton?

**Comment by rsmb7z** [comment]
@cjdsellers It seems there have been some changes on the Docker repository side, as this used to work before. I'll check it out and provide an update over the weekend.

**Comment by Xepheryy** [comment]
@rsmb7z I already have a working fix, its a really simple one. Should I create a PR ?

**Comment by cjdsellers** [comment]
Hi @Xepheryy

Absolutely, that would be much appreciated!

**Comment by Xepheryy** [comment]
Hi, sorry for the delay it seems like the simple fix has some regressions. I'll continue to investigate.

**Comment by rsmb7z** [comment]
Hi @Xepheryy,
I've tested the container, and it's working fine. The only recent change is that IB now requires Push Authentication for paper accounts, which wasn't the case before. While this enhances the realism by mimicking the real account experience, the issue arises if the authentication isn't completed before the notification expires—something that previously needed external handling for live accounts. Now that paper accounts also require authentication, the Docker container shouldn't be tightly coupled with Nautilus.

I've made some updates in #1940 to allow the option to load any custom image through config.
